packages/web/components/Roles.tsx

```diff
@@ -0,0 +1,52 @@
+import { MetaTag, SelectSearch, Tooltip, Wrap, WrapItem } from '@metafam/ds';
+import { PlayerRole } from 'graphql/autogen/types';
+import React from 'react';
+import { RoleOption } from 'utils/roleHelpers';
+
+export type SetupRolesProps = {
+  roleChoices: Array<PlayerRole>;
+  placeHolder: string;
+  roles: Array<RoleOption>;
+  setRoles: React.Dispatch<React.SetStateAction<Array<RoleOption>>>;
+  id?: string;
+};
+
+export const RolesSelect: React.FC<SetupRolesProps> = ({
+  roleChoices,
+  placeHolder,
+  roles,
+  setRoles,
+  id,
+}) => (
+  <SelectSearch
+    menuPlacement="top"
+    isMulti
+    value={roles}
+    onChange={(value) => setRoles(value as Array<RoleOption>)}
```

Comment:
Does this work?

```suggestion
    onChange={(values: Array<RoleOption>) => setRoles(values)}
```

Reply:
getting
```
Type '(values: Array<RoleOption>) => void' is not assignable to type '(value: ValueType<{ label: string; value: string; }>, action: ActionMeta<{ label: string; value: string; }>) => void'.
  Types of parameters 'values' and 'value' are incompatible.
    Type 'ValueType<{ label: string; value: string; }>' is not assignable to type 'RoleOption[]'.
      Type 'undefined' is not assignable to type 'RoleOption[]'.ts(2322)
Select.d.ts(169, 3): The expected type comes from property 'onChange' which is declared here on type 'IntrinsicAttributes & Props<{ label: string; value: string; }> & { children?: ReactNode; }'
```
when i try to make this change